View.elm

```diff
@@ -69,8 +69,8 @@ renderBoardSquare address position square =
           ] []
 
 
-renderBoard : Address Action -> Board -> Html
-renderBoard address board =
+renderBoard : Address Action -> Color -> GameState -> Board -> Html
```

Comment:
Since we're using `Color`, `GameState` and `Board`, we could pass the whole `Game`, which would make the signature shorter but increase coupling. I chose to keep it like that.